app/src/pages/experiment/ExperimentCompareListPage.tsx

```diff
@@ -740,6 +813,18 @@ export function ExperimentCompareListPage() {
     data: tableData,
     columns,
     getCoreRowModel: getCoreRowModel(),
+    columnResizeMode: "onChange",
+  });
+
+  const rows = table.getRowModel().rows;
+  const virtualizer = useVirtualizer({
+    count: rows.length,
+    getScrollElement: () => tableContainerRef.current,
+    estimateSize: () => {
+      const numExperiments = experiments.length;
+      return 25 + numExperiments * 25;
```

Comment:
25 is a magic number here. Best to explain it a bit in a comment or in constants.